control-base/src/main/java/fi/nls/oskari/control/data/GetPrintHandler.java

```diff
@@ -354,6 +360,23 @@ private PrintLayer findById(List<PrintLayer> layers, int id) {
         return arr;
     }
 
+    private PrintLayer createMyPlacesPrintLayer(int categoryId, LayerProperties requestedLayer) {
+        int opacity = requestedLayer.getOpacity() != null ? requestedLayer.getOpacity() : 100;
+        if (opacity <= 0) {
+            // Ignore fully transparent layers
+            return null;
+        }
+
+        PrintLayer layer = new PrintLayer();
+        layer.setId(categoryId);
+        layer.setType("myplaces");
+        layer.setVersion("1.3.0");
+        layer.setName("oskari:my_places_categories");
+        layer.setSrsName("EPSG:3067");
```

Comment:
You should take srs name from properties. There's something like oskari.native.srs that's used elsewhere in the codebase.

Reply:
Better yet I should just remove the line all together, as it makes no difference what the value is in this case. The value added to the proxied GetMap request on the `crs`/`srs` key is determined by the `PrintRequest` not by the `PrintLayer`.